Exercises-3-Basic/src/exercise/Number1.java

```diff
@@ -0,0 +1,19 @@
+package exercise;
+//Write a Java program to sum the values of an array.
+//Example,
+//Input: An array of integers [2 , 5 , 1, 4, 10]
+//Output: should be the sum of all the values: 2 + 5 + 1 + 4 + 10 = 22.
+
```

Comment:
No newline line here is needed.
But need newline after `package exercise;`.
Please make the change in all the files.